app/views/workers/index.html.slim

```diff
@@ -21,8 +21,8 @@ table
           = api_key.name
         td
           = api_key.key
-        td = link_to 'Edit', edit_api_key_path(api_key)
-        td = button_to 'Destroy', api_key, method: :delete
+        td = link_to 'Edit', edit_api_key_path(api_key), class: "button button--secondary"
```

Comment:
Could you please change the single quotes to double quotes here? Thanks homie ❤️